src/Cameras/camera.ts

```diff
@@ -506,17 +527,15 @@ export class Camera extends Node {
         var engine = this.getEngine();
 
         if (this.mode === Camera.PERSPECTIVE_CAMERA) {
-            check = this._cache.fov === this.fov
-                && this._cache.fovMode === this.fovMode
-                && this._cache.aspectRatio === engine.getAspectRatio(this);
-        }
-        else {
-            check = this._cache.orthoLeft === this.orthoLeft
-                && this._cache.orthoRight === this.orthoRight
-                && this._cache.orthoBottom === this.orthoBottom
-                && this._cache.orthoTop === this.orthoTop
-                && this._cache.renderWidth === engine.getRenderWidth()
-                && this._cache.renderHeight === engine.getRenderHeight();
+            check = this._cache.fov === this.fov && this._cache.fovMode === this.fovMode && this._cache.aspectRatio === engine.getAspectRatio(this);
```

Comment:
same

Reply:
this is done automatically by prettier when linting, maybe we should change the rules in .prettierrc so it doesn't mess that much with indentation ?

Reply:
Totally (adding our pretty man :) @RaananW )

Reply:
Or run the whole repo with the current rules to make it up to date and make everyone use prettier/those rules?

Reply:
I will pass over the entire repo very soon. probably next week. as it seems to be disturbing a lot

Reply:
Also, notice it didn't do it on the next one. it is related to the location of the "&&" characters. unifying this is not a bad thing.